.travis.yml

```diff
@@ -40,14 +40,19 @@ install:
         cd ..
         mkdir -p catkin_ws/src
         mv opw_kinematics catkin_ws/src/
-        cd catkin_ws
+        cd catkin_ws        
         choco uninstall mingw
+        wget https://raw.githubusercontent.com/johnwason/source_bat/master/source_bat.bash
+        source source_bat.bash 'C:\Program Files (x86)\Microsoft Visual Studio\2017\BuildTools\VC\Auxiliary\Build\vcvars64.bat'
         choco source add -n=ros-win -s="https://roswin.azurewebsites.net/api/v2" --priority=1
         choco install ros-melodic-ros_base -y --execution-timeout=0
-        wget https://gist.githubusercontent.com/Levi-Armstrong/e3bb008e579a99c52eb74235dff6735a/raw/travis_ci_windows.bat
-        cmd "/C call travis_ci_windows.bat install "
+        #wget https://gist.githubusercontent.com/Levi-Armstrong/e3bb008e579a99c52eb74235dff6735a/raw/travis_ci_windows.bat
+        #cmd "/C call travis_ci_windows.bat install "        
+        source source_bat.bash 'c:\opt\ros\melodic\x64\setup.bat'
```

Comment:
Looks good to me. Are you wanting to keep the commented outline before merging?

Reply:
Comments are removed